source/first-steps/install/getting-meshroom/platforms/google-colab.rst

```diff
@@ -0,0 +1,4 @@
+Google Colaboratory
+===================
+
+`<https://github.com/alicevision/meshroom/wiki/Meshroom-in-Google-Colab-(cloud)>`_
```

Comment:
Can be moved here from the wiki. Requires some clean up and a new tested 2020.1 colab notebook

Reply:
I agree, but first I think it would make sense to have an official notebook that way all efforts in this can be focused in one place and it can be documented properly because I did do some searching and there any many notebooks available with varying degrees of functionality.